shared/editor/nodes/OrderedList.ts

```diff
@@ -57,31 +81,94 @@ export default class OrderedList extends Node {
       wrappingInputRule(
         /^(\d+)\.\s$/,
         type,
-        (match) => ({ order: +match[1] }),
+        (match) => ({ order: +match[1], listStyle: "number" }),
         (match, node) => node.childCount + node.attrs.order === +match[1]
       ),
+
+      wrappingInputRule(
+        /^([a-z])\.\s$/,
+        type,
+        (match) => ({
+          order: match[1].charCodeAt(0) - 96,
+          listStyle: "lower-alpha",
+        }),
+        (match, node) => {
+          const expectedChar = String.fromCharCode(
+            96 + node.childCount + node.attrs.order
+          );
+          return expectedChar === match[1];
+        }
+      ),
+
+      wrappingInputRule(
+        /^([A-Z])\.\s$/,
+        type,
+        (match) => ({
+          order: match[1].charCodeAt(0) - 64,
+          listStyle: "upper-alpha",
+        }),
+        (match, node) => {
+          const expectedChar = String.fromCharCode(
+            64 + node.childCount + node.attrs.order
+          );
+          return expectedChar === match[1];
+        }
+      ),
     ];
   }
 
   toMarkdown(state: MarkdownSerializerState, node: ProsemirrorNode) {
     state.write("\n");
 
     const start = node.attrs.order !== undefined ? node.attrs.order : 1;
-    const maxW = `${start + node.childCount - 1}`.length;
-    const space = state.repeat(" ", maxW + 2);
 
-    state.renderList(node, space, (index: number) => {
-      const nStr = `${start + index}`;
-      return state.repeat(" ", maxW - nStr.length) + nStr + ". ";
-    });
+    if (node.attrs.listStyle && node.attrs.listStyle === "lower-alpha") {
+      const space = state.repeat(" ", 4);
+
+      state.renderList(node, space, (index: number) => {
+        const letterIndex = ((start + index - 1) % 26) + 1;
```

Comment:
While the `letterIndex` wraps around here, in css-land it actually doubles up `aa`:

<img width="312" height="333" alt="image" src="https://github.com/user-attachments/assets/cfcf9197-c682-46ae-a792-42c1cb3be796" />

Could we make this behavior match?

Reply:
sure, that make more sense.